projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -64,26 +65,42 @@ The initial values for the `home` and `away` properties of the state are 0.
 
 ### Creating the reducer function
 
-The reducer function's responsibility is to handle the state transitions in an immutable way. Define a reducer function that handles the actions for managing the state of the scoreboard.
```

Comment:
I can't highlight it but on line 15, we define the reducer function with `- The switch statement`

Reply:
Fixed